build/cmake/AddMCRL2Binary.cmake

```diff
@@ -175,38 +181,55 @@ function(_add_mcrl2_binary TARGET_NAME TARGET_TYPE)
      (IS_GUI_BINARY AND NOT MCRL2_ENABLE_GUI_TOOLS))
     return()
   endif()
+ 
+  get_filename_component(SOURCEDIR_ABS ${ARG_SOURCEDIR} ABSOLUTE)
+  get_filename_component(INCLUDEDIR_ABS ${ARG_INCLUDEDIR} ABSOLUTE)
 
-  foreach(SRC ${ARG_SOURCES})
-    get_filename_component(SRC_ABS ${SRC} ABSOLUTE)    
-    if(NOT "${SRC_ABS}" STREQUAL "${SRC}")
-      get_filename_component(SRC_ABS ${ARG_SOURCEDIR}/${SRC} ABSOLUTE)
+  foreach(_SRC ${ARG_SOURCES})
+    get_filename_component(_SRC_ABS ${_SRC} ABSOLUTE)
+
+    # TODO: Is this necessary/wanted?
```

Comment:
I can imagine that we just leave the old way of searching for `filename` and then `source_dir`/`filename`. This is also not very nice, but it seems like tools do rely on this behaviour.